src/model/Layer.ts

```diff
@@ -2,6 +2,10 @@ import { FeatureCollection } from 'geojson';
 import BaseEntity, { BaseEntityArgs } from './BaseEntity';
 import LayerType from './enum/LayerType';
 
+export interface DefaultRequestParams {
+  transparent?: boolean;
```

Comment:
As we don't exactly know, what is contained in the given params, the type should changed to something like:

```ts
[key?: string]: string | number | boolean;
```